CHANGES/5165.misc

```diff
@@ -0,0 +1 @@
+Sync collections using stages API
```

Comment:
I don't think users will understand this. Can the be restated as `5165.feature` and have it focus on that we can sync all collection (a full mirror) from Galaxy.